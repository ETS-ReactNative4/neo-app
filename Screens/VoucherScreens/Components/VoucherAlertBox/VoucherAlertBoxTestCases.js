import React from "react";
import VoucherAlertBox from "./VoucherAlertBox";

const VoucherAlertBoxTestCases = [
  {
    title: "Voucher Alert Box",
    Component: <VoucherAlertBox />
  }
];

export default VoucherAlertBoxTestCases;
