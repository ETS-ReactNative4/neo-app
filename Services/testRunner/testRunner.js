import renderer from "react-test-renderer";

const testRunner = testCase => {
  test(testCase.title, () => {
    const tree = renderer.create(testCase.Component).toJSON();
    expect(tree).toMatchSnapshot();
  });
};

export default testRunner;
