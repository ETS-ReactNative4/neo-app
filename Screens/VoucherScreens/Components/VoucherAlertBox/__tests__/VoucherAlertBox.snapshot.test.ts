import VoucherAlertBoxTestCases from "../VoucherAlertBoxTestCases";
import testRunner from "../../../../../Services/testRunner/testRunner";

VoucherAlertBoxTestCases.map(testRunner);
