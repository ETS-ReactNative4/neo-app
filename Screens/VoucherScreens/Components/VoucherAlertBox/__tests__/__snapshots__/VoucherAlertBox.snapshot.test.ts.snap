// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Voucher Alert Box - info mode 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "flex-start",
        "borderRadius": 4,
        "flexDirection": "row",
        "justifyContent": "flex-start",
        "padding": 24,
      },
      Object {
        "backgroundColor": "rgba(38, 119, 240, 0.3)",
      },
      Object {},
    ]
  }
>
  <View
    style={
      Object {
        "marginTop": 4,
      }
    }
  >
    <Unknown
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": "rgba(38, 119, 240, 1)",
            "fontSize": 14,
          },
          undefined,
          Object {
            "fontFamily": "voyager",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      
    </Unknown>
  </View>
  <View
    style={
      Object {
        "marginLeft": 8,
      }
    }
  >
    <Unknown
      style={
        Array [
          Object {
            "fontFamily": "SourceSansPro-Regular",
            "fontSize": 14,
            "lineHeight": 20,
          },
          Object {
            "color": "rgba(38, 119, 240, 1)",
          },
        ]
      }
    >
      As the transfer is on shared basis, the arrival time of transfer could vary plus or minus 15-30minutes depending on traffic
    </Unknown>
  </View>
</View>
`;

exports[`Voucher Alert Box 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "flex-start",
        "borderRadius": 4,
        "flexDirection": "row",
        "justifyContent": "flex-start",
        "padding": 24,
      },
      Object {
        "backgroundColor": "rgba(236, 71, 71, 0.3)",
      },
      Object {},
    ]
  }
>
  <View
    style={
      Object {
        "marginTop": 4,
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": "rgba(236, 71, 71, 1)",
            "fontSize": 14,
          },
          undefined,
          Object {
            "fontFamily": "voyager",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      
    </Text>
  </View>
  <View
    style={
      Object {
        "marginLeft": 8,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "fontFamily": "SourceSansPro-Regular",
            "fontSize": 14,
            "lineHeight": 20,
          },
          Object {
            "color": "rgba(236, 71, 71, 1)",
          },
        ]
      }
    >
      As the transfer is on shared basis, the arrival time of transfer could vary plus or minus 15-30minutes depending on traffic
    </Text>
  </View>
</View>
`;
